macros/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "geoengine-macros" # having a dash in the name breaks test runner codelense in rust analyzer
```

Comment:
comment is outdated

Reply:
[a112b49](https://github.com/geo-engine/geoengine/pull/859/commits/a112b49808b69cdee640aa1110103a0956cfaee2)